src/ray/exported/exported.cc

```diff
@@ -0,0 +1,34 @@
+#include "ray/core_worker/core_worker.h"
+#include "ray/exported/test.h"
+
+namespace ray {
+namespace exported_ {
+
+void SubmitActorTask(const ActorID &peer_actor_id,
```

Comment:
Can we name this differently? It's confusing that it has the same name as the CoreWorker method.

Reply:
Actually according to the streaming code below, it looks like this should be called `SendInternal`?

Reply:
renamed